packages/openapi-to-graphql/test/example_api_server.js

```diff
@@ -153,8 +153,8 @@ function startServer(PORT) {
 
   const Patents = {
     'CCC OSv1': {
-      patentId: '100',
```

Comment:
Typo in the example API. It does not match the OAS. It is already sanitized here.